charts/daps-reg-service/values.yaml

```diff
@@ -28,7 +28,7 @@ image:
   repository: "ghcr.io/catenax-ng/tx-daps-registration-service/dapsreg" 
   pullPolicy: Always
   # Overrides the image tag whose default is the chart appVersion.
-  tag: "1.0.4"
+  tag: "latest"
```

Comment:
Why using latest tag? It is not permitted (TRG 5.01, TRG 5.05).

Reply:
Changed